SOURCES/WEBAPP/ESP32/aurora/aurora.ino

```diff
@@ -1320,6 +1320,7 @@ void softMuteDAC( void )
  */
 void softUnmuteDAC( void )
 {
+  delay( 250 );
```

Comment:
If the delay is placed before the unmute command, doesn't it mean that the issue is in fact in the code executed before this call to softUnmuteDAC() ?
Do we know which code is executed just before this call.
Perhaps this is where the delay should be moved?

Reply:
As you can see in the bug, I have some pretty ugly gains for the PEQ, which I suppose causes some numerical artifact in the DSP; which takes a while to settle. I doesn't matter what parameter I change in the DSP - if these PEQ are enabled, the output clips hard.

I have no idea how to prevent that, except maybe not permitting insane PEQ settings to begin with. (OTOH those settings work well for me).

Reply:
Ah, it happens on any changes. I missed that point.
So it seems safe, without slowering things too much.

@dspverden what's your take on this?